dynamic-simulation/dynamic-simulation-tool/src/main/java/com/powsybl/dynamicsimulation/tool/DynamicSimulationTool.java

```diff
@@ -49,6 +52,7 @@
 
     private static final String CASE_FILE = "case-file";
     private static final String DYNAMIC_MODEL_FILE = "dynamic-model-file";
+    private static final String DYNAMIC_EVENT_MODEL_FILE = "events-file";
```

Comment:
Rename this constant EVENTS_MODEL_FILE, and the parameter `events-model-file` to be consistent

Reply:
Done